src/Reader/Extension/Atom/Feed.php

```diff
@@ -166,6 +166,7 @@ public function getDescription()
             return $this->data['description'];
         }
 
+        /** @psalm-suppress UnusedVariable */
```

Comment:
This variable **is** unused though?

Reply:
It **was** used, but Psalm wasn't seeing it. Regardless, I've rewritten the code, and the error is now gone.